cts_can/cts_can.py

```diff
@@ -558,26 +581,8 @@ def resetAll(bus, status_dict, cmd_dict):
 def initialise_can(cts):
```

Comment:
I think in the lines above `diffK'  is defined but never used and `applyDict` is not even defined. So `resetAll()` seems not to be in working order.

Reply:
that looks strange indeed, I will investigate it further.